tests/rom/test_spherical_rom.py

```diff
@@ -45,3 +45,17 @@ def test_spherical_rom_model():
         point = spherical_rom_model.sample_reachable_position(rng)
         distance = np.linalg.norm(point - sphere_center)
         assert distance < sphere_radius + 1e-6
+
+    # Test training the parameters.
+    init_params = spherical_rom_model.get_trainable_parameters()
+    assert np.isclose(init_params, sphere_radius)
+
+    data = [
+        (sphere_center + np.array([0.0, 0.0, 0.9]), False),
```

Comment:
Is the label here flipped? 0.9 should be positive -> True, 1.1 negative -> False